### ISSUE TITLE
For Banks section appears even when logging in 

### ISSUE THREAD

**Issue by simonredfern** [OpenBankProject/OBP-API#802]
This section should be hidden from login panel

**Comment by simonredfern** [comment]
<img width="1153" alt="for_banks_section" src="https://user-images.githubusercontent.com/485218/32275200-a3a8103c-bf0a-11e7-99ca-d4f8d34592bc.png">

**Comment by sebtesobe** [comment]
Of course! Sorry, fixed in a05b1b9d0b0c4c63137be8a27a56b0eb88102f35 . Also deployed to https://apisandbox.openbankproject.com

**Comment by simonredfern** [comment]
Cool!
